exporter/kineticaexporter/exporter_metric_test.go

```diff
@@ -0,0 +1,560 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package kineticaexporter
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.uber.org/zap/zaptest"
+)
+
+var testServer *httptest.Server
+var baseURL string
+
+// var showTableMetricGaugeResponse = "\x04OK\x00&show_table_response\xe2\x13\"otel.metric_gauge\x02\"otel.metric_gauge\x00\x02\x00\x00\x02$303100960528700881\x00\x02\xc2\x03{\"type\":\"record\",\"name\":\"type_name\",\"fields\":[{\"name\":\"gauge_id\",\"type\":\"string\"},{\"name\":\"metric_name\",\"type\":\"string\"},{\"name\":\"metric_description\",\"type\":[\"string\",\"null\"]},{\"name\":\"metric_unit\",\"type\":[\"string\",\"null\"]}]}\x00\x02\x00\x00\x02\x08\x10gauge_id\x08\x08data\x16primary_key\x12shard_key\x08uuid\x00$metric_description\x06\x08data\x0echar256\x10nullable\x00\x16metric_name\x04\x08data\x0echar256\x00\x16metric_unit\x06\x08data\x0echar256\x10nullable\x00\x00\x00\x028\"attribute_indexes\x00 collection_names\x08otel$compressed_columns\x000datasource_subscriptions\x00\x18foreign_keys\x00\"foreign_shard_key\x00$global_access_mode\x14read_write,is_automatic_partition\nfalse\x10is_dirty\x00\"is_view_persisted\x00\"last_refresh_time\x00(owner_resource_group<kinetica_system_resource_group*partition_definitions\x004partition_definitions_json\x04{}\x1cpartition_keys\x00\x1cpartition_type\x08NONE\x18record_bytes\x06784\x1crefresh_method\x00&remaining_table_ttl\x04-1\"request_avro_json\xda\x02{\"table_name\":\"otel.metric_gauge\",\"type_id\":\"303100960528700881\",\"options\":{\"foreign_keys\":\"\",\"is_replicated\":\"false\",\"is_result_table\":\"false\",\"no_error_if_exists\":\"true\"}}\"request_avro_type\x10is_table\x16schema_name\x08otel&strategy_definitionR( ( VRAM 1, RAM 5, DISK0 5, PERSIST 5 ) )\x1atable_monitor\x04{}\x12table_ttl\x04-1\x16total_bytes\x020\x1euser_chunk_size\x0e8000000\x1eview_table_name\x00\x00\x00\x00\x00\x00\x01\x01\x00\x00"
```

Comment:
Can these commented-out vars be removed?

Reply:
Yes, will remove them now that the tests have run

Reply:
Removed the commented vars from the test.